projects/frontend/data-pipelines/gui/projects/data-pipelines/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@vdk/data-pipelines",
+  "version": "0.0.1",
+  "engines": {
+    "node": "^14.15.0 || >=16.10.0",
+    "npm": ">=8.5.5"
+  },
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@angular/animations": "~13.3.10",
+    "@angular/cdk": "~13.3.8",
+    "@angular/common": "~13.3.10",
+    "@angular/compiler": "~13.3.10",
+    "@angular/core": "~13.3.10",
+    "@angular/forms": "~13.3.10",
+    "@angular/platform-browser": "~13.3.10",
+    "@angular/platform-browser-dynamic": "~13.3.10",
+    "@angular/router": "~13.3.10",
+    "@apollo/client": "~3.6.5",
+    "@auth0/angular-jwt": "~5.0.2",
+    "@cds/core": "~5.7.2",
+    "@clr/angular": "~13.3.1",
+    "@clr/icons": "~13.0.2",
+    "@clr/ui": "~13.3.1",
+    "@fingerprintjs/fingerprintjs": "~3.3.3",
+    "@ngrx/effects": "~13.2.0",
+    "@ngrx/router-store": "~13.2.0",
+    "@ngrx/store": "~13.2.0",
+    "@swimlane/ngx-charts": "~16.0.0",
+    "@swimlane/ngx-graph": "~8.0.0",
+    "@vdk/shared": "0.0.1",
+    "@webcomponents/custom-elements": "~1.5.0",
+    "@yellowspot/ng-truncate": "~2.0.1",
+    "angular-ng-autocomplete": "~2.0.8",
+    "apollo-angular": "~3.0.1",
+    "argparse": "~1.0.10",
+    "chart.js": "~3.7.1",
+    "chartjs-adapter-date-fns": "~2.0.0",
+    "chartjs-plugin-datalabels": "~2.0.0",
+    "chartjs-plugin-zoom": "~1.2.1",
+    "cron-parser": "~4.4.0",
+    "cronstrue": "~2.7.0",
+    "d3": "~5.16.0",
+    "date-fns": "~2.28.0",
+    "debug": "~4.3.4",
+    "dompurify": "~2.3.8",
+    "elasticlunr": "~0.9.5",
+    "file-saver": "~2.0.5",
+    "graphql": "~16.5.0",
+    "html-element": "~2.3.1",
+    "html-frontmatter": "~1.6.1",
+    "js-sha256": "~0.9.0",
+    "lodash": "~4.17.21",
+    "lottie-web": "~5.9.4",
+    "lz-string": "~1.4.4",
+    "moment": "~2.29.3",
+    "ng2-date-picker": "~12.0.4",
+    "ngx-clipboard": "~15.1.0",
+    "ngx-cookie-service": "~13.2.1",
+    "ngx-lottie": "~8.2.0",
+    "ngx-timeago": "~2.0.0",
+    "remove-html-comments": "~1.0.2",
+    "rxjs": "~7.5.5",
+    "striptags": "~3.2.0",
+    "superagent": "~5.3.1",
+    "ua-parser-js": "~1.0.2",
+    "zone.js": "~0.11.5"
```

Comment:
You should bump zone.js to at least 0.11.7 to avoid asynchronous tests failing.

https://github.com/angular/angular/issues/45476

Reply:
let's do lib upgrades afterwards (those could be functional changes)